modules/virt-cnv-network-alerts.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-events.html/virt-virtualization-alerts.adoc
+
+[id="virt-cnv-network-alerts_{context}"]
```

Comment:
Add the content type attribute as described in the Content type attribute requirements document. Here it would be:
:_content-type: PROCEDURE